src/sa_web/static/css/default.css

```diff
@@ -2090,3 +2121,142 @@ browser height, but in portrait mode, we usually have ~30px of nav/status-bar, s
+
+/* Multi-step place form */
+
+form.dev-place-form-choice {
+
+}
+form.dev-place-form-choice .toggle-sibling-input {
+  position: absolute;
+  left: -9999px;
+}
+form.dev-place-form-choice .toggle-sibling-label.button {
+  display: block;
+  float: left;
+  width: 40%;
+  font-size: 100%;
+  padding: 1em 1em;
+  margin: 0 2% 2% 0;
+  cursor:pointer;
+  background-color: #dff7ff;
+  color: #6c7180;
+}
+
+form.dev-place-form-choice .toggle-sibling-label.button img {
+  vertical-align: middle;
+  width: 2em;
+  margin-right: .5em
+}
+form.dev-place-form-choice .toggle-sibling-label.button strong {
+}
+
+form.dev-place-form-choice .toggle-sibling-label.button:hover {
+  color: white;
+}
+form.dev-place-form-choice .observation-label.button:hover {
+  background-color: #63c2c0;
+}
+form.dev-place-form-choice .question-label.button:hover {
+  background-color: #d5d72e;
+}
+form.dev-place-form-choice .idea-label.button:hover {
+  background-color: #fc9229;
+}
+form.dev-place-form-choice .complaint-label.button:hover {
+  background-color: #ff0000;
+}
+
+form.dev-place-form-choice input:checked+.toggle-sibling-label.button {
+  background-color: #008CBA;
+  color: #fff;
+}
+
+form.dev-place-form-choice input:checked+.observation-label.button {
+  background-color: #63c2c0;
+}
+form.dev-place-form-choice input:checked+.question-label.button {
+  background-color: #d5d72e;
+}
+form.dev-place-form-choice input:checked+.idea-label.button {
+  background-color: #fc9229;
+}
+form.dev-place-form-choice input:checked+.complaint-label.button {
+  background-color: #ff0000;
+}
+
+form.dev-place-choice-child-form {
+  display: none;
+}
+
+/* v2 Multi-step place form */
+
+#place-form-type-choice {
+
+}
+#place-form-type-choice a.button {
+  display: block;
+  float: left;
+  width: 40%;
+  font-size: 100%;
+  padding: 1em 1em;
+  margin: 0 2% 2% 0;
+  cursor:pointer;
+  background-color: #dff7ff;
+  color: #6c7180;
+}
+
+#place-form-type-choice a.button img {
+  vertical-align: middle;
+  width: 2em;
+  margin-right: .5em
+}
+#place-form-type-choice a.button strong {
+}
+
+#place-form-type-choice a.button:hover {
+  color: white;
+}
+#place-form-type-choice a.button#observation-form-type:hover {
+  background-color: #63c2c0;
+}
+#place-form-type-choice a.button#question-form-type:hover {
+  background-color: #d5d72e;
+}
+#place-form-type-choice a.button#idea-form-type:hover {
+  background-color: #fc9229;
+}
+#place-form-type-choice a.button#complaint-form-type:hover {
+  background-color: #ff0000;
+}
+
+#place-form-type-choice a.button.active {
+  background-color: #008CBA;
+  color: #fff;
+}
+
+#place-form-type-choice #observation-form-type a.button.active {
+  background-color: #63c2c0;
+}
+
+#place-form-type-choice #question-form-type a.button.active {
+  background-color: #d5d72e;
+}
+
+#place-form-type-choice #idea-form-type a.button.active {
+  background-color: #fc9229;
+}
+
+#place-form-type-choice #complaint-form-type a.button.active {
```

Comment:
We should consider moving these hard-coded styles for `idea`, `question`, `observation` and `complaint` into the config. On one hand, having CSS in the config isn't pretty, but having our config in the CSS could make things difficult. For example, if someone changes a place name in the config, it would break the styles.